### ISSUE TITLE
Auto generation of routers based on Partitions for Stateless services

### ISSUE THREAD

**Issue by Mahons** [dariopb/traefikServiceFabricPlugin#12]
I'd like to make the auto-generation of services based on partitions optional (or potenially disable it altogether) for Stateless services. 

@dariopb 

According to the Microsoft [documentation](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-concepts-partitioning):

"partitioning a stateless service is a very rare scenario--scalability and availability are normally achieved by adding more instances."

At the moment, I'm creating services with custom routers and the auto-generated routers are noisy in my configuration and something I will never use.

I'm happy to create a PR, however, @dariopb, as you're the owner, I'd like to discuss it with you first.

**Comment by dariopb** [comment]
I agree, we should define either a plugin-wide config parameter that controls the auto partition route behavior (like, generate all partition routes or only the first one) or could be a specific label that controls it per service.

**Comment by Mahons** [comment]
I think it's okay to leave it as is for stateful services, i.e., generate a route for each partition. However, for stateless, I'd be inclined to turn it off globally and provide a label to turn it on as required. I won't get a chance this week but will try get something together over the following week.

Do you want to assign this issue to me?

**Comment by dariopb** [comment]
@Mahons, would you like me to generate a new release with all the changes or are you using the plugin with via the local plugin way?

**Comment by Mahons** [comment]
@dariopb No harm creating a release; I'm using it via the localPlugin configuration, but there may be others who are using it via the pilot website.
